52/vipinreyo/pomodoro.py

```diff
@@ -0,0 +1,80 @@
+"""
+A simple command line based Pomodoro timer. Timer duration and break duration can be passed as arguments, but are
+optional. Default timer value is set as 25 minutes. Default break duration is set as 0 minutes.
+The timer is defined as a class for reuse.
+
+Note: Please pip install 'colorama' pacakge as this is used to color the output.
+"""
+
+from datetime import datetime, timedelta
+import time
+import sys
+import argparse
+from colorama import Fore
```

Comment:
next time might want to include a requirements.txt (no worries, I will)

Reply:
Sure. Will do.